packages/notification/docs/ToastNotification.stories.tsx

```diff
@@ -224,6 +224,20 @@ const StickerSheetTemplate: Story = () => {
       ),
       autohide: !IS_CHROMATIC,
     })
+    addToastNotification({
+      type: "positive",
+      title: "VerylongTitleExampleVerylongtitleExample ",
```

Comment:
Needs to be longer!
![image](https://user-images.githubusercontent.com/820635/220211774-b4c571fa-2e5e-484b-b2f2-1a526b5cd6ac.png)